frontend/pages/errors/Fleet500/Fleet500.tsx

```diff
@@ -36,7 +36,7 @@ const Fleet500 = () => (
       <p>Please file an issue if you believe this is a bug.</p>
       <div className={`${baseClass}__button-wrapper`}>
         <a
-          href="https://osquery.slack.com/join/shared_invite/zt-h29zm0gk-s2DBtGUTW4CFel0f0IjTEw#/"
+          href="https://fleetdm.com/support"
```

Comment:
@mikermcneil do we want to also change the associated button texts, for example line 50 "Get help on Slack" to be "Get support" or similar?

Reply:
cc @noahtalerman @rachaelshaw since Mike has more important things to do

Reply:
@jacobshandling Not at all!  Nothing more important than the user experience making sense.  I think we're good here-- talked it over a bit with @Sampfluger88 in the "CEO office minutes" Gong recording from Dec 26 (you can search for it in google drive), and then we tried to cut the video short so it's not too painful to watch

basically "specific, AND inclusive is worth bait-and-switch risk" is probably the best I've got right now to sum up this principle